middleware.js

```diff
@@ -19,9 +19,10 @@ var imports = {};
  *                   when undefined defaults to `src`
  *    `root`        A base path for both source and destination directories
  *    `outputStyle` Sass output style (nested or compressed), nested by default
+ *    `indentedSyntax` Use standard SCSS sytax (Sassy CSS) or the cleaner SASS syntax
  *    `force`       Always re-compile
  *    `debug`       Output debugging information
- *    `response`    True (default) to write output directly to response 
```

Comment:
Yup, but it is the right change.
README should also be synchronized with the recent changes.